sparkle/src/components/ElementDialog.tsx

```diff
@@ -33,6 +34,8 @@ export function ElementDialog<T>({
       onCancel={() => onCancel(transitionOnClose)}
       onValidate={() => onValidate(transitionOnClose)}
       {...props}
-    />
+    >
+      {children}
```

Comment:
Why this change?

Reply:
My linter was actually warning `Element Dialog doesn't have required attribute children`